Integrations/MISP_V2/MISP_V2.yml

```diff
@@ -41,6 +41,19 @@ configuration:
   - Value
   required: false
   type: 16
+- additionalinfo: Include Related events for Events, if not selected all will be retreived.
+  display: Retreive Related events for Events
+  hidden: false
+  name: related_events
+  required: false
+  type: 8
+- additionalinfo: Include the Galaxies and tags of the matching attributes, if not
+    selected all will be retreived.
+  display: Retreive Galaxys and Tags for Events
```

Comment:
this one looks redundent with context_select
Think you should just add defaults to context_select, and leave out all the array fields.